src/config/config.rs

```diff
@@ -40,19 +40,49 @@ pub struct Config {
 #[derive(Debug, Parser)]
 /// Storage config
 pub struct StorageConfig {
-    #[clap(long = "storage-type", value_name = "VALUE", default_value = "none")]
-    /// Storage type: S3,None
+    #[clap(long = "storage-type", value_name = "VALUE", default_value = "fs")]
+    /// Storage type: s3, fs
     pub storage_type: String,
-    #[clap(
-        long = "storage-cache-capacity",
-        value_name = "VALUE",
-        default_value_t = 1073741824
-    )]
-    /// Set memory cache capacity, default is 1GB
-    pub cache_capacity: usize,
+    #[clap(flatten)]
+    /// The memory cache config
+    pub memory_cache_config: MemoryCacheConfig,
     #[clap(flatten)]
     /// S3 storage config
     pub s3_storage_config: S3StorageConfig,
+    #[clap(long = "storage-fs-root", value_name = "VALUE", default_value_t)]
+    /// The root of FS backend
+    pub fs_storage_root: String,
+}
+
+/// Memory cache config
+#[derive(Debug, Parser)]
+pub struct MemoryCacheConfig {
+    /// The capacity of memory cache in blocks, default is 16384 blocks.
+    #[clap(
+        long = "storage-mem-cache-capacity",
+        value_name = "VALUE",
+        default_value_t = 16384
+    )]
+    pub capacity: usize,
```

Comment:
Use the size of memory instead of the number of stored blocks.

Reply:
Should we also provide an item to config the `block_size`? Or just leave it as a constant for now?

Reply:
provide with default value.